bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,321 +1,835 @@
+async function CreateRecurringOfficeHourswithCustomTimezone({
+  dataStore,
+  client,
+  actionParameters,
+}) {
+  const startDate = moment.utc().subtract(1, "m").format();
+
+  const responseOfSwitchCoustomTimeZone = await client.fetch(`v1.0/me/events`, {
+    method: "POST",
+    body: JSON.stringify({
+      subject: actionParameters.subject,
+      body: {
+        contentType: "HTML",
+        content: actionParameters.content,
+      },
+      start: {
+        dateTime: `${actionParameters.startDate}T${actionParameters.startTime}`,
+        timeZone: actionParameters.timezone,
+      },
+      end: {
+        dateTime: `${actionParameters.endDate}T${actionParameters.endTime}`,
+        timeZone: actionParameters.timezone,
+      },
+      allowNewTimeProposals: true,
+      isOnlineMeeting: actionParameters.isOnlineMeeting,
+      onlineMeetingProvider: "teamsForBusiness",
+      recurrence: {
+        pattern: {
+          type: actionParameters.recurrencetype,
+          interval: 1,
+          daysOfWeek: [actionParameters.days],
+          dayOfMonth: actionParameters.dayOfMonth,
+        },
+        range: {
+          type: "endDate",
+          startDate: moment(actionParameters.startDate).format("YYYY-MM-DD"),
+          endDate: actionParameters.recurEndDate,
+        },
+      },
+    }),
+  });
+  console.log(JSON.stringify(responseOfSwitchCoustomTimeZone));
+  if (!responseOfSwitchCoustomTimeZone.ok) {
+    throw new Error(
+      ` Could not create Event (${responseOfSwitchCoustomTimeZone.status}: ${responseOfSwitchCoustomTimeZone.statusText})`
+    );
+  }
+  const user_Id = [actionParameters.userId];
+  // console.log(user_Id);
+  await calendarView(client, dataStore, user_Id, startDate);
+  await myEvents(client, dataStore, user_Id);
+}
+//EditOneTimeWithCurrentTimezone
+async function EditOneTimeWithCurrentTimezone({
+  dataStore,
+  client,
+  actionParameters,
+}) {
+  const startDate = moment.utc().subtract(1, "m").format();
+
+  // console.log(startDate)
+  const responseOfCurrentTimeZone = await client.fetch(
+    `v1.0/me/events/${actionParameters.id}`,
+    {
+      method: "PATCH",
+      body: JSON.stringify({
+        subject: actionParameters.subject,
+        body: {
+          contentType: "HTML",
+          content: actionParameters.content,
+        },
+        start: {
+          dateTime: actionParameters.startDateTime,
+          timeZone: actionParameters.timezone,
+        },
+
+        end: {
+          dateTime: actionParameters.endDateTime,
+          timeZone: actionParameters.timezone,
+        },
+        location: {
+          displayName: actionParameters.location,
+        },
+        attendees: [
+          {
+            emailAddress: {
+              address: actionParameters.email1,
+            },
+            type: actionParameters.type1,
+          },
+          {
+            emailAddress: {
+              address: actionParameters.emai2,
```

Comment:
Here it is `address: actionParameters.emai2` but in the action parameters schema it is `email2`. Name of the variable is mismatched.
Is this script really tested? If yes, then this should be results in error.